applications/matter_bridge/src/bridged_device_types/onoff_light_switch.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
+
+#include "matter_bridged_device.h"
+
+class OnOffLightSwitchDevice : public MatterBridgedDevice {
+public:
+	static constexpr uint16_t kOnOffClusterRevision = 4;
+	static constexpr uint32_t kOnOffFeatureMap = 1;
+	static constexpr uint16_t kBindingClusterRevision = 1;
+	static constexpr uint32_t kBindingFeatureMap = 0;
```

Comment:
Can we maybe read these values from Matter cluster definitions? I guess they are defined in some zap-generated files.

Reply:
Well I would love to, but unfortunately I don't think it's possible. We have tried to do that in other bridge files as well, but the info is generated in the following form: https://github.com/nrfconnect/sdk-nrf/blob/main/applications/matter_bridge/src/zap-generated/endpoint_config.h#L74 So we would need a parser to extract specific data from GENERATED_ATTRIBUTES, which is sort of done in the Matter attribute storage, but the methods are private and moreover cannot parse it on the compilation level.

Reply:
Warm greetings from the ZAP world.